.github/workflows/main.yml

```diff
@@ -76,10 +76,28 @@ jobs:
         run: |
           git fetch --tags
           echo "tag=$(git describe --tags --abbrev=0)" >> $GITHUB_ENV
+      - name: Import BOT GPG key
+        run: echo $BOT_GPG_KEY | base64 --decode | gpg --batch --import
```

Comment:
This ensures the key is never persisted outside of the encrypted Github Actions secrets, or the GPG agent.